src/Symfony/Component/Serializer/Normalizer/VaryingSupportInterface.php

```diff
@@ -12,15 +12,16 @@
 namespace Symfony\Component\Serializer\Normalizer;
```

Comment:
I've removed any mention about caching, because that is an implementation detail of what the serializer does with classes implementing this interface. I think it's better to have a pure abstraction here.